guidelines/php/drupal/README.md

```diff
@@ -0,0 +1,111 @@
+Writing PHP in Drupal 8
+=======================
+
+We follow the [Drupal Coding standards](https://www.drupal.org/docs/develop/standards).
+For our Drupal 8 projects we have a separate
+[How-do-we-drupal-article](https://github.com/gridonic/drupal8).
+
+When you read our global guidelines carefully you find the line:
+
+> We use 4 spaces for indentation
+
+In our drupal projects we will ignore the 4 spaces in all php- & twig-files and
+we use just 2 spaces for indentation.
+
+## Exceptions
+
+We adhere to most rules for Drupal project, but add some exceptions where we
+disagree or find the original Drupal rules not suitable for our use case(s).
+These exceptions are listed below.
+
+### Comments
+
+A default-comment based on the guidelines looks like this:
+
+```php
+/**
+ * Short Title.
+ *
+ * Longer description (optional).
+ *
+ * @param string $value_a
+ *   Description of @param A.
+ * @param string $value_b
+ *   Description of @param B.
+ *
+ * @return array
+ *   Description of @return.
+ */
+public function exampleFunction($value_a, $value_b) {
+    $concat_values = [];
+    
+    $concat_values[] = $value_a;
+    $concat_values[] = $value_b;
+    
+    return $concat_values;
+}
+```
+
+In most of the functions this makes sense and we use this. But from time to
+time part of the comment is really clear.
+
+You can write the same comment without the descriptions:
+
+```php
+/**
+ * @param string $value_a
+ * @param string $value_b
+ *
+ * @return array
+ */
+```
+
+But if you wanna add descriptions, they have to be in the correct way:
```

Comment:
Maybe shortly describe the required format of the comments somewhere, in addition to the excellent examples:

> But if you ~~wanna~~ want to add comments, they need to respect the following format:

* Every comment must end with a period
* The short title of a function must not be longer than 80 characters
* The optional longer description of a function must be separated by a blank line from the sort title
* Descriptions of parameters go to the next line and are indented with 2 spaces

Reply:
This would be really helpful, since I'm always struggling with those rules!

Reply:
Thanks for your additions, i have updated the README with your inputs.